src/server/response.cpp

```diff
@@ -155,7 +155,7 @@ HTTPErrorResponse::HTTPErrorResponse(const InternalServer& server,
   : ContentResponseBlueprint(&server,
                              &request,
                              httpStatusCode,
-                             request.get_requested_format() == "html" ? "text/html; charset=utf-8" : "application/xml; charset=utf-8",
+                             request.get_requested_format() == "html" ? "text/html; charset=utf-8" : "application/xml; charset=utf-8 ;raw=true",
```

Comment:
Though this change addresses my earlier comment in an explicit way, it is actually not needed